docs/source/getting_started/estimate_likelihood.ipynb

```diff
@@ -0,0 +1,324 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def ordered_logit_loglikeobs(params, y, x):\n",
+    "    \"\"\"Likelihood function of an orderd logit model.\"\"\"\n",
+    "    # parse the parameter vector into its quantities\n",
+    "    beta = params.loc[\"beta\", \"value\"].to_numpy()\n",
+    "    cutoffs = params.loc[\"cutoff\", \"value\"].to_numpy()\n",
+    "    \n",
+    "    # calculate deterministic part of utilities\n",
+    "    xb = x.dot(beta)\n",
+    "    \n",
+    "    # evaluate likelihood\n",
+    "    upper_cutoffs = np.hstack([cutoffs, np.inf])[y]\n",
+    "    lower_cutoffs = np.hstack([-np.inf, cutoffs])[y]\n",
+    "    upper_cdf = stats.logistic.cdf(upper_cutoffs - xb)\n",
+    "    lower_cdf = stats.logistic.cdf(lower_cutoffs - xb)\n",
+    "    \n",
+    "    return np.log(upper_cdf - lower_cdf)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### The ordered_logit command\n",
+    "\n",
+    "The estimation command follows the same patter as the ``maximize`` or ``minimize``. Thus, nothing has to be changed except for the criterion function to adopt the other interface."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def ordered_logit(formula, data, dashboard=False):\n",
+    "    \"\"\"Estimate an ordered probit model with maximum likelihood.\n",
+    "    \n",
+    "    Args:\n",
+    "        formula (str): A patsy formula\n",
+    "        data (str): A pandas DataFrame\n",
+    "        dashboard (bool): Switch on the dashboard.\n",
+    "        \n",
+    "    Returns:\n",
+    "        res: optimization result.\n",
+    "        \n",
+    "    \"\"\"\n",
+    "    params, y, x, constr = ordered_logit_processing(formula, data)\n",
+    "    \n",
+    "    res = estimate_likelihood(\n",
+    "        criterion=ordered_logit_loglikeobs, \n",
+    "        params=params, \n",
+    "        algorithm='scipy_L-BFGS-B',\n",
+    "        constraints=constr,\n",
+    "        criterion_kwargs={\"y\": y, \"x\": x}, \n",
+    "        dashboard=dashboard)\n",
+    "    return res"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that ``estimate_likelihood`` automatically uses the ``maximize`` function, so you don't have to switch the sign of your objective function!\n",
+    "\n",
+    "### Use the command"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.read_pickle(\"ologit.pickle\")\n",
+    "form = \"apply ~ pared + public + gpa\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "info, params = ordered_logit(form, df, dashboard=False)"
```

Comment:
This does not run on my machine. The error is a KeyError in `estimate_likelihood` in line 81. `result` is already a dictionary with an entry "fitness".

Reply:
Fixed.